### ISSUE TITLE
Add "Accessibility statement"/"Deklaracja dostępności" page

### ISSUE THREAD

**Issue by wfenrich** [CeON/fairchive#1276]
As a website run by a public entity, we are required by law to have it, so we should have it. It should work very similar to privacy policy page:

1. Some installations will have it, some will not (having it is not a worldwide requirement). 
2. It will contain a styled text, probably with headers and some oredered/undoredered lists.
2. We may wish to have a separate text for each interface language. 
3. The text itself should be WCAG compliant (so it should break after 80 pages). 
4. If the page exists, it should be listed in the sitemap.

**Comment by wfenrich** [comment]
Already made in https://github.com/CeON/dataverse/issues/863
